the colored blocked are designed without any margin or padding

Soroushsp opened the pull request:
https://soroushsp.github.io/layout_snake/
https://soroushsp.github.io/layout_snake/report/html_report/

Soroushsp commented:
https://soroushsp.github.io/layout_snake/
https://soroushsp.github.io/layout_snake/report/html_report/

Soroushsp commented:
[DEMO LINK](https://soroushsp.github.io/layout_snake/)
[TEST REPORT LINK](https://soroushsp.github.io/layout_snake/report/html_report/)

ihor-jpeg reviewed (changes requested):
your snake has to be responsive. setting the width here is a no-go

![Screenshot 2022-08-23 at 17 02 25](https://user-images.githubusercontent.com/79282749/186178523-0adc03c5-43a4-432d-8656-2d8a48126e41.jpg)

Soroushsp commented:
I am sorry I didn't know you need something adaptive at any dimension.
Now blocks are responsive and you can see they occupy the whole space that viewport provides horizontally. 
for example at the 802x600 pixels blocks are exactly 300px height and they are 401 pixels wide each. 

also there was a margin issue due to default 16px margin exerted by browsers which I fixed that 

also I did something very smart for you, I removed all unnecessary CSS codes, now the results are produced by 100 lines 
instead of 160 lines 

I hope you like this task, it costed me many hours to made this. 

[DEMO LINK](https://soroushsp.github.io/layout_snake/)
[TEST REPORT LINK](https://soroushsp.github.io/layout_snake/report/html_report/)
